resources/backend/js/gutenberg.js

```diff
@@ -169,6 +172,60 @@ function convertKitGutenbergRegisterBlock( block ) {
 					);
 					break;
 
+				case 'resource':
+					// Build options for <select> input.
+					fieldOptions.push(
+						{
+							label: '(None)',
+							value: '',
+						}
+					);
+					for ( let value in field.values ) {
```

Comment:
I think for...of would be better here since for...in has some wonky behavior ([SO](https://stackoverflow.com/questions/500504/why-is-using-for-in-for-array-iteration-a-bad-idea)). And I'm pretty sure you can use const here vs. let.

Reply:
Good spot. Have updated, thanks!